tests/basic/upgrade_test.go

```diff
@@ -504,3 +506,196 @@ var _ = Describe("{UpgradeVolumeDriverFromCatalog}", func() {
 		AfterEachTest(contexts)
 	})
 })
+
+var _ = Describe("{UpgradePxKvdbMemberDown}", func() {
+	/*
+					https://purestorage.atlassian.net/browse/PTX-21450
+				    https://portworx.testrail.net/index.php?/cases/view/94371
+					1. Bring down a kvdb member node
+			        2. Upgrade PX
+			        3. Bring up the kvdb member node
+		            4. Verify if the kvdb member joins the cluster and everything is running fine
+	*/
+
```

Comment:
Please add JustBeforeEach and JustAfterEach.

Reply:
Done.